WcaOnRails/spec/factories/inbox_persons.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :inbox_person do
+    id { 1 }
```

Comment:
You overestimate our "awesome" legacy db structure:
```sql
CREATE TABLE `InboxPersons` (
  `id` varchar(10) COLLATE utf8mb4_unicode_ci NOT NULL,
  `wcaId` varchar(10) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
[...]
```

Yep it probably would break, any suggestion on how to make it somehow autoincrement for a given competition?
The simplest may be to make it mandatory for the caller to pass the person id.

Reply:
Not efficient, but probably working way would be something like:
```rb
id { InboxPerson.maximum(:id) + 1 }
```

Reply:
Right!
Also ids don't have to be unique, just the tuple `(id, competitionId)`.